src/app/docs/toolbar/toolbar-docs.component.html

```diff
@@ -0,0 +1,156 @@
+<sky-docs-demo-page
+  moduleName="SkyToolbarModule"
+  moduleSourceCodePath="src/app/public/modules/toolbar/"
+  pageTitle="Toolbar"
+>
+  <sky-docs-demo-page-summary>
+    The toolbar component displays actions for lists, reocrds, and tiles.
+  </sky-docs-demo-page-summary>
+
+  <sky-docs-design-guidelines>
+    <sky-docs-demo-page-section
+      heading="Usage"
+    >
+      <sky-docs-design-guideline
+        heading="Use when"
+      >
+        <p>
+          Use list toolbars to display actions that manipulate the contents of lists of records. The list toolbar appears above a <a href="https://developer.blackbaud.com/skyux/components/list">list</a> of records. Place frequently used actions directly in the toolbar, and place less-frequently used actions in the more actions menu. When multiple views are available, dock the view switcher on the right.
```

Comment:
I don't think we need the second sentence. We could just make `lists` in the first sentence the link to the list docs and then delete the second sentence altogether.